src/js_native_api_types.h

```diff
@@ -115,4 +115,22 @@ typedef struct {
   napi_status error_code;
 } napi_extended_error_info;
 
+typedef enum {
+  napi_key_all,
+  napi_key_own
+} napi_key_collection_mode;
+
+typedef enum {
+  napi_key_only_writable = 1,
+  napi_key_only_enumerable = 1 << 1,
+  napi_key_configurable = 1 << 2,
```

Comment:
Those three, `napi_key_only_writable`, `napi_key_only_enumerable` and `napi_key_configurable` have similar semantic meanings that narrow down the criteria of the key queries, yet not named in similar way.

Would it be possible to name them as `napi_key_writable` and `napi_key_enumerable`?